webapp/crashstats/documentation/views.py

```diff
@@ -236,7 +236,13 @@ def generate_field_doc(dataset, field):
             search_field_query_type = super_search_field["query_type"]
             # FIXME(willkg): we're only doing this for public fields, but we could do
             # this for whatever fields the user can see
-            if field_data["permissions"] == ["public"]:
+            if field_data["permissions"] == ["public"] and (
+                # text fields only get example data if they have a full version or
+                # include fielddata, like for semicolon_keywords fields.
+                super_search_field["has_full_version"]
+                or super_search_field["storage_mapping"]["type"] != "text"
```

Comment:
this will also work for es 1.4 because there the type is "string" and does support aggregations